### ISSUE TITLE
Can't approve hashtag trends on mobile

### ISSUE THREAD

**Issue by twilde** [mastodon/mastodon#21432]
### Steps to reproduce the problem

1. Open the trending hashtags UI on mobile (Chrome iOS)
2. No checkboxes or approve/reject UI appear

### Expected behaviour

Checkboxes and approve/reject UI appear

### Actual behaviour

No checkboxes or approve/reject UI appear

### Specifications

Mastodon 4.0.2 (docker)
Chrome 107.0.5304.101
iOS 16.1.1

**Comment by twilde** [comment]
Screenshot: ![4C26502B-16DB-4C74-83AC-DBCD5EE191B0](https://user-images.githubusercontent.com/1042669/203433080-915eda1c-ce4a-4de7-ad1a-cf6e15de453a.png)

**Comment by twilde** [comment]
Nope, no visible change when switching to request desktop site (even after a forced reload).

**Comment by ineffyble** [comment]
Okay, not just an issue of spacing then (I thought the checkboxes might have been pushed off-screen).

**Comment by smiba** [comment]
The checkboxes disappear when your screen is too small, the same happens if you make your desktop browser window (too) small.

Most definitely an UI issue, the checkboxes should persist regardless of mobile or desktop layout.
